samples/msal-browser-samples/VanillaJSTestApp2.0/app/facebook-sample/authConfig.js

```diff
@@ -0,0 +1,71 @@
+// Config object to be passed to Msal on creation
+const msalConfig = {
+    auth: {
+        clientId: "ENTER_CLIENT_ID_HERE",
+        authority: "https://www.facebook.com",
+        knownAuthorities: ["www.facebook.com"],
+        protocolMode: msal.ProtocolMode.OIDC,
+        OIDCOptions: { "serverResponseType": "query", "defaultScopes": ["openid"] },
```

Comment:
Can we show enum usage?
```suggestion
        OIDCOptions: { "serverResponseType": msal.ServerResponseType.QUERY, "defaultScopes": ["openid"] },
```